src/porepy/geometry/intersections.py

```diff
@@ -1527,8 +1554,9 @@ def _point_in_or_on_polygon(p: np.ndarray, poly: np.ndarray, tol=1e-8):
         return 2, None
 
 
-@pp.time_logger(sections=module_sections)
-def triangulations(p_1, p_2, t_1, t_2):
+def triangulations(
```

Comment:
Could the function name be made more specific?

Reply:
The idea is that you would always call it as pp.intersections.triangulations, which should be sufficiently explicit (and long)

Reply:
Ok!